src/main/c/seasocks/Connection.h

```diff
@@ -70,6 +70,8 @@ class Connection : public WebSocket {
     virtual void send(const char* webSocketResponse) override;
     virtual void send(const uint8_t* webSocketResponse, size_t length) override;
     virtual void close() override;
+    virtual void setUserdata(std::shared_ptr<void> udata) override { _userdata = udata; }
+    virtual std::shared_ptr<void> getUserdata() const override { return _userdata; }
```

Comment:
I'd prefer the capitalization `setUserData` etc to fit in with the other things

Reply:
Yeah sure, I was not really sure about caps as I usually write_stuff_with_underscores_instead.